docs/configuration.rst

```diff
@@ -33,7 +33,7 @@ PostgreSQL default is to use ASCII. Either change your client settings,
 or specify the encoding by appending ``?client_encoding=utf8`` to the
 connection string. This will look like::
 
-    SQLALCHEMY_DATABASE_URI = 'postgresql://myuser:mypass@localhost/dbname?client_encoding=utf8'
+    SQLALCHEMY_DATABASE_URI = 'postgresql://myuser:mypass@localhost/mydb?client_encoding=utf8'
```

Comment:
Is there any rationale for that change? In this case, the example for MariaDB should be adapted in the same way.

Reply:
Because it matches this:
https://github.com/spiral-project/ihatemoney/blob/6bacae1cd9d136e99a818b0e7f4f1f829803ed88/docs/installation.rst#L101

I know the intention is for the user to use their own names, but in practice they may not. And `myuser` and `mypass` match already, so we may as well be consistent.